src/bencode.c

```diff
@@ -26,45 +26,45 @@
 
 #include "config.h"
 
-#include <stdlib.h>             /* malloc() realloc() free() strtoll() */
-#include <string.h>             /* memset() */
 #include <ctype.h>
+#include <stdlib.h> /* malloc() realloc() free() strtoll() */
+#include <string.h> /* memset() */
 
 #include <glib.h>
 
 #include "bencode.h"
 
-static be_node *be_alloc(be_type type)
+static be_node* be_alloc(be_type type)
 {
-    be_node *ret = g_malloc0(sizeof(be_node));
+    be_node* ret = g_malloc0(sizeof(be_node));
```

Comment:
This style is very uncommon in C codebases, none within the glib ecosystem. It's more of a C++ thing IME.